### PR TITLE
Added Typings for bootstrap validator

### PR DESCRIPTION
**Pull request by BradyNadeau**
Please fill in this template.

- [X - Pulled Types2.0] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [X] Run `tsc` without errors.
- [X] Run `npm run lint package-name` if a `tslint.json` is present.

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not provide its own types, and you can not add them.
- [X] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [X] Create it with `npm run new-package package-name`, not by basing it on an existing project.

If changing an existing definition:
- [N/A] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [N/A] Increase the version number in the header if appropriate.

If removing a declaration:
- [N/A] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [N/A] Delete the package's directory.
- [N/A] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: bootstrap-validator/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    feedback?: any;
+    custom?: any;
+}
+
+interface JQuery {
+    validator(options?: ValidatorOptions): JQuery;
+    validator(command: string): JQuery;
+}
+
+declare module "bootstrap-validator" {
```
Comment (andy-ms) [inline bootstrap-validator/index.d.ts]: If `bootstrap-validator` is just a jquery plugin that has no exports of its own, remove this.
If it has exports, describe them.

File: bootstrap-validator/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": false,
```
Comment (andy-ms) [inline bootstrap-validator/tsconfig.json]: `true` for both of these

**Review by ghost — CHANGES_REQUESTED** [review]
Needs `tslint.json` containing `{ "extends": "../tslint.json" }`.
We no longer accept pull requests for `types-2.0`. (#13233) Try `git rebase --onto master types-2.0`. (Be sure to make a backup of your changes first.) Then hit "Edit" in the upper-right of this page and change the base branch to `master`.

File: bootstrap-validator/index.d.ts (hunk 1 above)
Comment (BradyLiles) [inline bootstrap-validator/index.d.ts]: Are you talking about line 22?

Comment (andy-ms) [inline bootstrap-validator/index.d.ts]: Yes, the one with the `declare module`.

**Pull request closed without merging**